debian/debathena-nsswitch-config.postinst

```diff
@@ -22,9 +22,16 @@ set -e
 
 case "$1" in
     configure)
-	if [ -x /var/lib/dpkg/info/libnss-mdns.postinst ]; then
-	    /var/lib/dpkg/info/libnss-mdns.postinst configure
-	fi
+	for postinst in \
+	    $(ldconfig -p \
+		  | sed -n 's/^\s*libnss_.*=> \(.*\.so\.2\)$/\1/p' \
```

Comment:
Why do we only care about .so.2?

Reply:
2 is the [only supported NSS interface version](https://www.gnu.org/software/libc/manual/html_node/Adding-another-Service-to-NSS.html). The real point was exclude bare `.so` symlinks found in `-dev` packages; I guess I could have said that more directly.